docs/Bitbucket.md

```diff
@@ -0,0 +1,169 @@
+> The Bitbucket Pipeline configuration that is described in the various sections below is specified in the central `bitbucket-pipelines.yml` within a specific Bitbucket workspace [repository](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). In Buildkite - pipeline configuration can be set in a singular `pipeline.yml` within a repository - but can also be set and uploaded dynamically through the use of [Dynamic Pipelines](https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines). Additionally, control and governance of Buildkite pipelines can be achieved by the use of [Pipeline Templates](https://buildkite.com/docs/pipelines/templates) to set shared pipeline configuration within a Buildkite organization.
+
+## Clone (`clone`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `clone` | ⚠️ | Clone options for all steps of a Bitbucket pipeline. The majority of these options should be set on a Buildkite agent itself via [configuration](https://buildkite.com/docs/agent/v3/configuration) of properties such as the clone flags (`git-clone-flags`, `git-clone-mirror-flags` if utilising a Git mirror), fetch flags (`git-fetch-flags`) - or changing the entire checkout process in a customised [plugin](https://buildkite.com/docs/plugins/writing) overriding the default agent `checkout` hook. <br/><br/> Sparse-checkout properties of `code-mode`, `enabled` and `patterns` will be translated to the respective properties within the [sparse-checkout-buildkite-plugin](https://github.com/buildkite-plugins/sparse-checkout-buildkite-plugin). <br/><br/> clone` properties at the Bitbucket pipeline, if set, have higher precedence over these global properties.  |
+
+## Definitions (`definitions`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `definitions` | ⚠️ | Customised definitions utilised throughout a Bitbucket pipeline. `caches` and `services` are supported for translation within Buildkite Migration tool. |
+| `definitions.caches` | ⚠️ | Customised cache definitions that can be applied to specific Bitbucket pipeline steps - inclusive of folders, single file - or multifile cache. Targeted into specific steps with the `pipelines.default.step.caches.<name>` property, and in which the translation will utilise the [cache-buildkite-plugin](https://github.com/buildkite-plugins/cache-buildkite-plugin) that may require further setup/tweaking around specific caching strategies. |
+| `definitions.caches.<name>` | ✅ | A customised cache name applicable for one or more steps within a Bitbucket pipeline. |
+| `definitions.caches.<name>.path` | ✅ | The directory path that is desired to be cached. |
+| `definitions.caches.<name>.key.files` | ⚠️ | The list (one or more) files that are monitored for changes - and stored once its hash changes between file versions change. If multiple files are specified - multiple cache-plugin definitions are set on the resultant Buildkite command step (differing `manifest` properties between each). |
```

Comment:
I would add a clarification stating that the multiple-files to multiple-cache plugin entries is not only is not equivalent, but also may probably cause issues due to multiple caches trying to manage the same folder.

Reply:
Added 👍